data/remnant/remnant.txt

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2017 by Michael Zahniser
+# Copyright (c) 2022 by williaji
```

Comment:
Just move this (and copy the GNU blurb) over to the commodities file, and I think this should be ready to go .